gramps/gui/filters/sidebar/_citationsidebarfilter.py

```diff
@@ -78,7 +78,7 @@ def __init__(self, dbstate, uistate, clicked):
         for conf_value in sorted(conf_strings.keys()):
             model.append((_(conf_strings[conf_value]),))
         self.filter_conf.set_model(model)
-        self.filter_conf.set_active(Citation.CONF_NORMAL)
+        self.filter_conf.set_active(Citation.CONF_VERY_LOW)
```

Comment:
With Kari's PR, we can cancel/close both PRs 1700 and 1701.

Also, I do not believe that relabeling the Confidences to harmonize with 5.5.1 GEDCOM confidence labels would be a good choice at this point. GEDCOM7 is discussing the addition of negative Confidence options (disproven & similar levels). So it would be better to do a single change that encompasses both labeling and scope changes.

Reply:
OK.  I'll close #1700 and #1701.

We can leave the confidence level labels for now.  The Gedcom text is already in the tooltips.